interface-elements-for-web/articles/card-view.md

```diff
@@ -23,5 +23,6 @@ This section describes the capabilities provided by the Card View control, which
 
```

Comment:
@AnnaKondratova it seems that a link to the "Exporting" section is missing here (and in toc.yaml too).

Reply:
Modified.